config/database.yml

```diff
@@ -14,13 +14,21 @@ default: &default
 development:
   <<: *default
   database: pancake_development
+  username: root
+  password: root
+  host: db
+  port: 5432
 
 # Warning: The database defined as "test" will be erased and
 # re-generated from your development database pancakeen you run "rake".
 # Do not set this db to the same as development or production.
 test:
   <<: *default
   database: pancake_test
+  username: root
+  password: root
+  host: db
+  port: 5432
```

Comment:
i don't think you should commit this.
You can add `DATABASE_URL` to your env, and then put it until default to use on dev and test.